client/src/CMakeLists.txt

```diff
@@ -8,3 +8,12 @@ set (SRC
 )
 
 add_executable(${TARGET} ${SRC})
+
+# target_include_directories(${TARGET}_core
+#     PUBLIC ${INCLUDE_DIR}
+#     PRIVATE ${SRC_DIR}
+# )
```

Comment:
?

Reply:
It's a piece of code that will work with some source files but won't without.
I can remove it if you want but it will be re-added soon
cc @osvegn @LucasTesnier